client/web/src/insights/InsightsRouter.tsx

```diff
@@ -71,7 +71,14 @@ export const InsightsRouter = withAuthenticatedUser<InsightsRouterProps>(props =
                 )}
             />
 
-            {codeInsightsDashboards && <Route path={`${match.url}/dashboard`} render={() => <DashboardsLazyPage />} />}
+            {codeInsightsDashboards && (
+                <Route
+                    path={`${match.url}/dashboard/:id?`}
```

Comment:
To be consistent with `EditInsightPage` route:

```suggestion
                    path={`${match.url}/dashboard/:dashboardID?`}
```